src/main/java/com/pollistics/models/Poll.java

```diff
@@ -16,22 +16,46 @@
 	private String name;
 	private HashMap<String,Integer> options;
 	private User user;
+	private String slug;
+
+	// todo: orden these constructors with `this(...arg)`
```

Comment:
Someone has an idea to make the constructors easier, they are so many for all the different kinds of unit tests 😢